app/presenters/users/sessions/new_presenter.rb

```diff
@@ -24,7 +24,13 @@ def school_name
       end
 
       def oauth_host
-        @oauth_host ||= Rails.application.secrets.sso_domain
+        return @oauth_host if @oauth_host
+
+        if Rails.env.development?
+          @oauth_host = Rails.application.secrets.sso_domain + ':3000'
```

Comment:
This should not be necessary - use nginx (as described in dev's docs) or puma-dev to setup your local dev environment.